dask/dataframe/io/parquet/arrow.py

```diff
@@ -61,6 +64,37 @@ def _get_row_groups_per_piece(pieces, metadata, path, fs):
     return tuple(result.values())
 
 
+def _merge_statistics(stats, s):
+    """ Update `stats` with vaules in `s`
+    """
+    stats[-1]["total_byte_size"] += s["total_byte_size"]
+    stats[-1]["num-rows"] += s["num-rows"]
+    ncols = len(stats[-1]["columns"])
+    ncols_n = len(s["columns"])
+    if ncols != ncols_n:
+        raise ValueError(f"Column count not equal ({ncols} vs {ncols_n})")
+    for i in range(ncols):
+        name = stats[-1]["columns"][i]["name"]
+        j = i
+        for ii in range(ncols):
+            if name == s["columns"][j]["name"]:
+                break
+            if ii == ncols - 1:
+                raise KeyError(f"Column statistics missing for {name}")
+            j = (j + 1) % ncols
+
+        min_n = s["columns"][j]["min"]
```

Comment:
It has recently been brought to my attention that max/min are [deprecated](https://github.com/apache/parquet-format/blob/master/src/main/thrift/parquet.thrift#L199) and the recommendation is to use max/min_value, which are the same except maybe reversed if the column_ordering value says so. I don't know why they chose this duplication! In any case, the code should probably look for the min/max_value first and fall back to min/max.

Reply:
Interesting - Thanks for the heads up!

Reply:
Just spent a bit of time looking through the pyarrow code. As far as I can tell, we are still expected to query `column.statistics.min` and `column.statistics.max` from the python API. 
Key names in the underlying thrift format is only important in pyarrow's c++ backend (the check for "min/max" vs "min/max_value" being [here](https://github.com/apache/arrow/blob/1de2593e0ce0cac3358936b69c8d1d13528e27b8/cpp/src/parquet/metadata.cc#L88), for example).   Please do let me know if I am confusing the logic - Otherwise, I will assume that there is no need to check for a `min`/`max_value` attribute that we know is not defined in pyarrow.

Reply:
OK, then - good that they have abstracted this away. fastparquet still has to worry about it...